corgidrp/mocks.py

```diff
@@ -84,18 +139,28 @@ def create_default_headers():
     prihdr = fits.Header()
     exthdr = fits.Header()
 
+    if obstype != "SCI":
+        NAXIS1 = 2200
+        NAXIS2 = 1200
+    else:
+        NAXIS1 = 2200
+        NAXIS2 = 2200
+
     # fill in prihdr
     prihdr['OBSID'] = 0
     prihdr['BUILD'] = 0
-    prihdr['OBSTYPE'] = 'SCI'
+    prihdr['OBSTYPE'] = obstype
     prihdr['MOCK'] = True
 
     # fill in exthdr
+    exthdr['NAXIS'] = 2
+    exthdr['NAXIS1'] = NAXIS1
+    exthdr['NAXIS2'] = NAXIS2
     exthdr['PCOUNT'] = 0
     exthdr['GCOUNT'] = 1
     exthdr['BSCALE'] = 1
     exthdr['BZERO'] = 32768
-    exthdr['ARRTYPE'] = 'SCI'
+    exthdr['ARRTYPE'] = obstype # seems to be the same as OBSTYPE
     exthdr['SCTSRT'] = '2024-01-01T12:00:00.000Z'
```

Comment:
I'm not sure what some of these headers are, but I assume they were all discussed and worked out already.  Otherwise, I approve of this pull request.  Given my ignorance of the headers here, I will let @semaphoreP or @maxwellmb pull the trigger to pull this in.  I made issues about the peripheral unresolved things (HIERARCH, obstypes, and simulated data that sometimes causes failed tests) that can be dealt with in another PR.

Reply:
Thanks for putting those issues in. These headers are all specified in our team confluence (see "Data Formats" page). I prescribed dummy values to each of them, so we should change the values when we need them to be more reasonable.